### ISSUE TITLE
Final Project Review

### ISSUE THREAD

**Issue by voganrc** [mihirparadkar/world-development#10]
Overall your project is very good. You do a good job of describing the features in the dataset, but I would have like to see some more exploratory data analysis showing how you first started to approach and analyze the dataset. You mention that you do a ~50% training/test split, but it would be nice to see you use k-fold cross validation to get more accurate error estimates considering the dataset is fairly small.

Nice job using GLRM's. You use two different regularizers, to get a point of comparison, and I think that the graph-based approach of linking years was particularly clever. It would be nice to see you try out some different loss functions other than huber though., just as a point of comparison. Also, it would be interesting to see some unsupervised methods of filling in the missing data, just to compare them to the predictions made by your model.

Bonus points for working out the math for the proximal gradient method.  Especially since you could have just made a function call that showed an added depth of understanding. Also good work on searching over the optimal hyperparameters, but again it would be nice to see the tuning expanded a bit with some other methods.
